netlify-plugin-cloudinary/src/lib/cloudinary.ts

```diff
@@ -186,20 +184,31 @@ export async function getCloudinaryUrl(options: CloudinaryOptions) {
     if (canSignUpload) {
       // We need an API Key and Secret to use signed uploading
 
-      results = await cloudinary.uploader.upload(fullPath, {
-        ...uploadOptions,
-      })
+      try {
+
+        results = await cloudinary.uploader.upload(fullPath, {
+          ...uploadOptions,
+        })
+      } catch (error) {
+        console.error(ERROR_ASSET_UPLOAD, error)
```

Comment:
may make sense for this to be 2 separate `console.error` statements for readability?